geoips/interfaces/yaml_based/sectors.py

```diff
@@ -91,7 +91,10 @@ def create_test_plot(self, fname):
         )
         ax.add_feature(cfeature.COASTLINE)
         ax.add_feature(cfeature.BORDERS)
-        fig.savefig(fname)
+        if fname is not None:
+            fig.savefig(fname)
+        if return_fig_ax_map:
+            return fig, ax, mapobj
```

Comment:
How does this get used? Would it make sense to just always return fig, ax, and mapobj? I know that calling `savefig` kind of destroys the objects, but I don't like conditionally returning things.

I'm mostly leaving this as a note to create an issue.

Reply:
See #566.